### PR TITLE
Added NoIP example to sample-etc_ddclient.conf

### PR DESCRIPTION
**Pull request by hgducharme**
I added a NoIP example to the `sample-etc_ddclient.conf` file in the same order that is listed for the protocols in the `README.md`.

### CODE REVIEW

**Comment by erul92** [comment]
{
  "date" : "2017-11-01 15:28:43 +0000",
  "client" : {
    "cver" : "12.42.10",
    "cmodel" : "iPhone5,1",
    "cosver" : "10.3.3.14G60",
    "cbr" : "com.google.ios.youtube"
  },
  "gfeedback" : {
    "e" : "11201412,11203231,11204247,11204533,11204628,23701018,23702322,23702700,23704012,23704248,23705361,23705485,23705739,23705960,23706129,23706602,23706964,23708199,23708772,23708791,23708886,23708904,23708906,23708910,23709706,23710230,23710762,23710969,9405989,9407610,9422596,9431754,9437138,9449243,9452749,9463575,9464090,9467146,9476026,9476327,9477113,9477962,9480475,9480495,9484238,9484345,9484688,9485000,9487591,9488038,9488369",
    "country-type" : "B",
    "logged_in" : "1"
  },
  "player" : "none",
  "cpn" : "none",
  "videoid" : "none"
}

File: sample-etc_ddclient.conf [hunk 1]
```diff
@@ -177,6 +177,14 @@ ssl=yes					# use ssl-support.  Works with
 # my.domain.tld,other.domain.tld
 
 ##
+## NoIP (noip.com)
+##
+# protocol=noip
+# server=dynupdate.no-ip.com
+# login=your-noip-login
```
Comment (erul92) [inline sample-etc_ddclient.conf]: # /etc/ddclient/ddclient.conf
#
protocol=dyndns2
use=web
login=mylogin
password=mypassword
myhost.dyndns.org

Comment (erul92) [reply sample-etc_ddclient.conf]: /usr/sbin/ddclient -daemon 300

File: sample-etc_ddclient.conf [hunk 2]
```diff
@@ -177,6 +177,14 @@ ssl=yes					# use ssl-support.  Works with
 # my.domain.tld,other.domain.tld
 
 ##
+## NoIP (noip.com)
+##
+# protocol=noip
```
Comment (fmarier) [inline sample-etc_ddclient.conf]: I'd suggest also including `ssl=yes` here so that the password isn't sent in the clear.

Comment (wimpunk) [reply sample-etc_ddclient.conf]: @fmarier Does noip supports ssl?

Comment (fmarier) [reply sample-etc_ddclient.conf]: Yes, here's the blurb I'm using:
```
$ sudo cat /etc/ddclient.conf
ssl=yes
protocol=noip
use=web, web=https://check.torproject.org/, web-skip='appears to be'
server=dynupdate.no-ip.com
login=username
password='password123'
hostname.example.com
```

**Pull request closed without merging**